src/commonlib/include/commonlib/coreboot_tables.h

```diff
@@ -87,6 +87,7 @@ enum {
 	LB_TAG_TYPE_C_INFO		= 0x0042,
 	LB_TAG_ACPI_RSDP		= 0x0043,
 	LB_TAG_PCIE			= 0x0044,
+	LB_TAG_CAPSULE			= 0x0045,
 	LB_TAG_EFI_FW_INFO		= 0x0046,
```

Comment:
Since these PRs have been reordered, it would make sense to change those tag numbers as well.

Reply:
Wanted to say that will will require changing the value in EDK and saw that it was `0xb0` there (used before making changes suitable for upstreaming) which needs to be changed anyway.  Updated: https://github.com/Dasharo/coreboot/compare/7a9776ca0d59889e3f63d565304a7942f15985a3..11de72c3ff2f755a0fb7ac45d38f93c5bf7a68c9#diff-8d6b99381f9485f5fbb941a0c702bad47ed392ba3058ede998847776409d37d2